### PR TITLE
Add open animation to carousel popup

### PR DESCRIPTION
**Pull request by foxwellm**
Add open animation to carousel popup, now slides open rather than just appear. Changed iconClassName to switch statement. Fixed bug, popup was not opening when not clicking directly on image.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class App extends Component {
 
     const popUpInfo = this.state.games
       .find(game => {
-        return game.img === event.target.src;
+        return game.game === event.target.closest('div').innerText;
```
Comment (foxwellm) [inline src/App.js:47]: PopUp was not opening when clicking on anywhere but image.

File: src/Components/GameCard.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ export default class extends Component {
     return (
       <div key={game.game} className="carousel-game">
         <div className="game-card" onClick={createPopUp}>
-          <span>{game.game}</span>
+          <span className="game-card-header">{game.game}</span>
           <img className="carousel-image" src={game.img} alt="game board"/>
```
Comment (foxwellm) [inline src/Components/GameCard.js:14]: Changed to a class name rather than using just span

File: src/Components/PopUp.js [hunk 3]
```diff
@@ -3,24 +3,27 @@ import React, { Component } from 'react';
...
     const { isSearch, game, closePopUp } = this.props;
-    let className = "PopUp";
-    let iconClassName = "fas fa-thermometer-half"
+    let className;
+    let iconClassName;
 
-    if (isSearch) {
-      className = "search-popup"
-    }
+    isSearch ? className = "search-popup" : className = "PopUp"
```
Comment (foxwellm) [inline src/Components/PopUp.js:10]: Changed to ternary for PopUp class names

File: src/Components/PopUp.js [hunk 4]
```diff
@@ -3,24 +3,27 @@ import React, { Component } from 'react';
...
+    switch (game.challenge_level) {
+      case 'simple':
+        iconClassName = "fas fa-thermometer-quarter"
+        break;
+      case 'average':
+        iconClassName = "fas fa-thermometer-half"
+        break;
+      default:
+        iconClassName = "fas fa-thermometer-full"
     }
```
Comment (foxwellm) [inline src/Components/PopUp.js:20]: Changed to switch statement for iconClassName

File: src/styles/_PopUp.scss [hunk 5]
```diff
@@ -1,8 +1,19 @@
...
+
+@keyframes openPopUp {
+  0% {
+    height: 0px;
+  }
+
+  100% {
+    height: 500px;
+  }
 }
```
Comment (foxwellm) [inline src/styles/_PopUp.scss:17]: Animation for popup opening

**Review by JustinD85 — APPROVED** [review]

**Review by whitneyfirth — APPROVED** [review]

**Pull request merged**